datamodel/builder.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2017 Camptocamp SA
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+"""
+
+Datamodels Builder
+==================
+
+Build the datamodels at the build of a registry.
+
+"""
+import odoo
+from odoo import api, models
```

Comment:
```suggestion
from odoo import api, models, modules
```
You can therefore remove import odoo